haiku-driver.groovy

```diff
@@ -128,14 +131,17 @@ def setLevel(level, duration) {
 }
 
 def sendLightLevelCommand(level) {
-    if (level > 16) {
-        level = 16
+    if (level > 100) {
+        level = 100
     }
     if (level < 0) {
         level = 0
     }
+    
+    Integer haikuLevel = Math.round(level / 6.25)
```

Comment:
The issue with rounding is that 1-3 get rounded down to zero , and it would be surprising to someone for 3% light level to mean "off". This is surprising too for higher numbers where the percentage has the effect of being "pulled down" to a lower level than you might expect.

I think a better way would be to use `Math.ceil(level / 6.25)`

This way:
0 -> 0
1-6 -> 1 (6 values)
7-12 -> 2 (6 values)
13-18 -> 3 (6 values)
19-25 -> 4 (7 values)
26-31 -> 5 (6 values)
...

This is a smooth step-up every 6 (or 7 in four cases) percentage points.

Let me know what you think of this!

Reply:
I did consider ceil() and floor() but settled on round() as it returned an integer while the others retun a double. ceil() would be a better implementation. I'll make that change.